app/models/rental.rb

```diff
@@ -39,7 +40,7 @@ class Rental < ActiveRecord::Base
   def renter_is_assignable
     return unless renter && creator # will be validated elsewhere, dont add an error twice
     if creator.assignable_renters.exclude? renter
-      errors.add :renter, 'must have permission to assign'
+      errors.add :creator, 'must have permission to assign'
```

Comment:
Shouldn't this still be renter? The renter is the one that it looks like the error is activating on.

Reply:
I actually interpreted this to mean "check that the person renting out this golf cart has permission to rent out this golf cart to this person" in which case 'renter' would not make sense here. Want me to go into detail on how I interpreted it?

Reply:
My only concern is the usefulness of the error. If we get this error, will it be easier to debug when it's added to the renter of the creator?

Reply:
Well, since the method checks the permissions of the creator and the error mentions someone having permissions it really makes absolutely zero sense to have the renter's name here. The creator is assigning the rental to the renter, so "renter must have permission to assign" would not in this case be helpful.
The error message could be better, of course, because this has zero user-friendliness. If Jon King sees this message, he's going to come to IT. We'll know what's happening, but no user would.
Now, an error message like "sorry, you only have permission to assign rentals to users in your department" would be much clearer, but that isn't the point of this PR.